index.php

```diff
@@ -1,48 +1,25 @@
-<html>
-<head>
-
-    <link rel="stylesheet" type="text/css" href="styles.css">
-    
-    <?php
-
-    $db = new PDO('mysql:host=db; dbname=mailmerge', 'root', 'password');
-
-$query = $db->prepare("SELECT `name`, `email` FROM `users`;");
-
-$query->setFetchMode(PDO::FETCH_ASSOC);
-
-$query->execute();
-
-$result = $query->fetchAll();
-
-?>
-
-</head>
-
-<body>
-    <header>
-        <h1>
-        What birds are outside my window?
-        </h1>
-    </header>
-
-    <section >
-        <p class="item">
-            <img src="https://www.transparentpng.com/thumb/seagull/Vph8Vo-seagull-free-download-transparent.png" alt="Seagull">
-        </item>
-
-        <p class="item">
-
-        </item>
-
-        <p class="item">
-
-        </item>
-
-
-    </section>
-</body>
-
+<!doctype html>
+<html lang='en'>
+    <head>
+        <link href="styles/normalize.css" rel="stylesheet" type="text/css">
+        <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
+        <link rel="stylesheet" type="text/css" href="styles.css">
+        <title>birds and that</title>
+        <?php require "display.php";
+              require "get.php";?>
```

Comment:
Would suggest moving these requires to the top of the file, it makes very little difference but standard convention is to have them at the top so you can quickly see if anything is required/included.